management_interface/management_interface/configuration.py

```diff
@@ -0,0 +1,18 @@
+import os
+from dataclasses import dataclass
+from typing import Union
+
+
+@dataclass(frozen=True)
+class Settings:
+    POSTGRES_DB: str = os.environ.get("HANS_MI_PSQL_DB", "postgres")
```

Comment:
Yes, the main (and only?) reason is for local development to avoid clash with other projects' settings. I like the env variable to be a little more verbose. And you're right, the name of this variable was initially `PSQL_DB`.

Reply:
I don't think this is relevant any more?  If we're moving to .env for local dev we can have whatever names we want, right?

Reply:
We can change this if we must. I'll resolve my own local environment vars then.